packages/hoppscotch-data/src/rest/v/1.ts

```diff
@@ -0,0 +1,219 @@
+import { defineVersion } from "verzod"
+import { z } from "zod"
+import { V0_SCHEMA } from "./0"
+
+export const FormDataKeyValue = z.object({
+  key: z.string(),
+  active: z.boolean()
+}).and(
+  z.union([
+    z.object({
+      isFile: z.literal(true),
+      value: z.array(z.instanceof(Blob))
+    }),
+    z.object({
+      isFile: z.literal(false),
+      value: z.string()
+    })
```

Comment:
Is this a good candidate for `discriminatedUnion` since there's a 
shared key` isFile`?